aws-msk/alerts/static-alert.yml

```diff
@@ -0,0 +1,71 @@
+---
+
+# Name of the alert
+name: Name of this alert condition
```

Comment:
Same comment on this alert!

Reply:
I had only the dashboard to submit. You may remove the alert config

Reply:
Thanks!